pyramid-rep-template

```diff
@@ -1,6 +1,6 @@
 <!--
 This HTML is auto-generated.  DO NOT EDIT THIS FILE!  If you are writing a new
-REP, see http://ros.org/rep/rep-0001 for instructions and links
+REP, see http://ros.org/rep/rep-0001.html for instructions and links
```

Comment:
This link doesn't work. It seems to lack a `s` (`/rep/` -> `/reps/`).

Reply:
fixed